doc/preparing.rst

```diff
@@ -0,0 +1,98 @@
+.. _preparing_repositories:
+
+Preparing a repository for Binder
+=================================
+
+You can do this by making sure your repo contains:
+
+* A collection of Jupyter Notebooks. These notebooks will be made available to
+  users of your Binder.
+* One (or many) text files that specify the requirements of your code. For
+  example, a ``requirements.txt`` or ``environment.yml`` file. See the
+  below examples for a list of all the files and environments that are
+  supported.
+
+All you need to do is include the files specified above in a GitHub repository.
+Once that's done, navigate to ``mybinder.org`` and insert the URL for
+your git repository. Press ``Launch`` to automatically create your Binder.
+The Binder service will be automatically send you to a live Jupyter session
+connected to this repository.
+
+.. note::
+
+   If a previous version of the repository has already been built, Binder will
+   only build a new one if the git hashes don't match. If Binder *doesn't* have
+   to build a new repository, the process of connecting to the live
+   computational environment is much faster.
+
+
+Simple python dependencies
+--------------------------
+
+A lot of repostories already contain a ``requirements.txt`` specifying the
+dependencies of that repository. If you only need on some simple to install
```

Comment:
For 'simple to install' dependencies, a ``requirements.txt`` should meet your needs.